js/common.js

```diff
@@ -1,4 +1,19 @@
+//Scroll to top Button
+window.onscroll = function() {scrollFunction()};
 
+function scrollFunction() {
+  if (document.body.scrollTop > 300 || document.documentElement.scrollTop > 300) {
+    document.getElementById("btnScrollToTop").style.display = "block";
+  } else {
+    document.getElementById("btnScrollToTop").style.display = "none";
```

Comment:
you can add document.getElementById("btnScrollToTop") into a variable and use it into both places.